e2e/flows/authentication.flow.ts

```diff
@@ -14,6 +14,7 @@ export class AuthenticationFlow {
             iaConfig.TestCaseOfficerUserName,
             iaConfig.TestCaseOfficerPassword
         );
+        await this.idamSignInPage.contentContains('Case List');
```

Comment:
I know it works, but it feels wrong to check this against `idamSignInPage`
Can we introduce `anyPage` and check via that so it makes better sense when reading?

Reply:
That's a fair point, I'd been thinking similar but had fallen on the side of "well, it works". But I'll add that in 👍